exercises/strings/rotN/rotN.js

```diff
@@ -15,15 +15,37 @@
  * @returns {string} A lowercase copy of the input string
  */
 
-function rotN(string) {
+function rotN(string, n) {
   // This is your job. :)
+  var i;
```

Comment:
No need to declare variables up top with JavaScript. You can declare + initialize variables at any point in the code (unlike in C/C++).